README.md

```diff
@@ -151,12 +151,21 @@ Sets a shared web credential. Resolves to `true` when successful.
 
 Inquire if the type of local authentication policy is supported on this device with the device settings the user chose. Should be used in combination with `accessControl` option in the setter functions. Resolves to `true` if supported.
 
-### `getSupportedBiometryType()`
+### `getSupportedBiometryType()` _(deprecated)_
```

Comment:
The reason this is deprecated is that it sometimes returns Face recognition as a supported biometric, even though it cannot be used for crypto operations.